helion/runtime/precompile_shim.py

```diff
@@ -25,7 +25,7 @@ def _make_precompiler(*args: object, **kwargs: object) -> Callable[[], None]:
         kwargs["debug"] = (
             kwargs.get("debug", fn.debug) or os.environ.get("TRITON_DEBUG", "0") == "1"
         )
-        kernel_cache, target, backend, binder = fn.device_caches[device]
+        kernel_cache, _, target, backend, binder = fn.device_caches[device]
```

Comment:
```suggestion
        kernel_cache, *_, target, backend, binder = fn.device_caches[device]
```
I think this will match either 0 or 1 things, so both versions of Triton will work (rather than breaking support for old ones).

Reply:
Nice, thanks :)